components/assets/icons/fontawesome-free-5.15.2-web/css/all.css

```diff
@@ -0,0 +1,4619 @@
+/*!
+ * Font Awesome Free 5.15.2 by @fontawesome - https://fontawesome.com
+ * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+ */
+.fa,
```

Comment:
I have a suggestion, fa need to be changed to more descriptive name, currently it does not convey any meaning